colorcoder/MinorColor.java

```diff
@@ -0,0 +1,25 @@
+package colorcoder;
+
+public enum MinorColor {
+    BLUE(0),
+    ORANGE(1),
+    GREEN(2),
+    BROWN(3),
+    SLATE(4);
+
+     public int index;
+        MinorColor(int index) {
+            this.index = index;
+        }
+        int getIndex() {
+            return index;
+        }
+        public static MinorColor fromIndex(int index) {
```

Comment:
This is almost identical to the function in `MajorColor`. Can you think of a way to reduce the code-duplication?